data/processed/trademarks/total_trademarks_scientific_2.yaml

```diff
@@ -0,0 +1,28 @@
+api_doc_url: https://www.gov.uk/government/publications/ipo-trade-mark-data-release/ipo-data-explained-2016
```

Comment:
What about this file `_2`? Please delete the undesired one and remove `_2` if needed